docs/release/updating-to-osg-36.md

```diff
@@ -21,9 +21,38 @@ yum update
 
 AI (Carl): https://opensciencegrid.atlassian.net/browse/SOFTWARE-4473
 
-#### osg-configure ####
+#### OSG-Configure ####
+
+The OSG 3.6 release series contains OSG-Configure 4, a major version upgrade from the previously released versions in the OSG.
+See the [OSG-Configure 4.0.0 release notes](https://github.com/opensciencegrid/osg-configure/releases/tag/v4.0.0)
+for an overview of the changes.
+Several configuration modules and options were removed or deprecated and CE configuration has been simplified;
+the update from version 3 to version 4 will require some manual changes to your configuration.
+
+To update OSG-Configure, perform the following steps:
+
+1.  Update the osg-configure packages:
+
+        :::console
+        root@host# yum update 'osg-configure*'
```

Comment:
You can drop this step as I plan on having a catch-all RPM update section (https://github.com/opensciencegrid/docs/pull/786/files#diff-56ab23f60058662c854a569e806dba7443ce99223a41cbb317825952fb6cf920R14)